pkg/apis/rbac/types.go

```diff
@@ -143,8 +140,8 @@ type ClusterRoleBinding struct {
 	// Subjects holds references to the objects the role applies to.
 	Subjects []Subject
 
-	// RoleRef can only reference the current namespace and the global namespace
-	// If the ClusterRoleRef cannot be resolved, the Authorizer must return an error.
+	// RoleRef can only reference a ClusterRole in the global namespace.
+	// If the RoleRef cannot be resolved, the Authorizer must return an error.
 	RoleRef api.ObjectReference
```

Comment:
@deads2k does it ever make sense for a ClusterRoleBinding to reference a Role in a namespace? If it was allowed would that Role's policies apply to the namespace the role is a part of?